cms/templates/visibility_editor.html

```diff
@@ -98,7 +115,12 @@ <h3 class="modal-section-title visibility-header" id="visibility-title">
                             % if group["deleted"]:
                             <label for="visibility-group-${partition["id"]}-${group["id"]}" class="label">
                                 ${_("Deleted Group")}
-                                <span class="note">${_('This group no longer exists. Choose another group or do not restrict access to this component.')}</span>
+                                <span class="note">
+                                    ## Translators: {block_type} specifies whether the xblock is a unit or a component
+                                    ${_('This group no longer exists. Choose another group or do not restrict access to this {block_type}.').format(
```

Comment:
@catong in this case (which is very rare), would you be OK with a simplification to say the following for both cases--

"This group no longer exists. Choose another group or remove the access restriction."

or

"This group no longer exists. Choose another group or do not restrict access."

?